src/definitions/modules/dropdown.js

```diff
@@ -2774,16 +2778,20 @@ $.fn.dropdown = function(parameters) {
                       module.save.remoteData(selectedText, selectedValue);
                     }
                     if(settings.useLabels) {
-                      module.add.label(selectedValue, selectedText, shouldAnimate);
                       module.add.value(selectedValue, selectedText, $selected);
```

Comment:
Could you please explain, why setting the value is still done even if the item is actionable, but setting is label/text will be prevented? Shouldn't even the value not being updated then?